### PR TITLE
 change pointer on mouse over on question types while adding new question. #5810 

### LINKED ISSUE #5810: change pointer on mouse over on question types while adding new question.

### PR DESCRIPTION
**Pull request by tarun14110**
Fixes #5810 

Before
screenshot from 2016-07-09 14 01 22

After
screenshot from 2016-07-09 13 54 47

### CODE REVIEW

**Comment by damithc** [comment]
Screenshot of before and after?

**Comment by tarun14110** [comment]
added

**Comment by kanghj** [comment]
@YongJieYongJie, can you review this since you are working on this page too?

File: src/main/webapp/js/instructorFeedbackEdit.js [hunk 1]
```diff
@@ -63,6 +63,10 @@ function readyFeedbackEditPage() {
         }
     });
     
+    $('#add-new-question-dropdown > li').mouseover(function() {
+        $(this).css('cursor', 'pointer');
+    });
+
```
Comment (YongJieYongJie) [inline src/main/webapp/js/instructorFeedbackEdit.js]: I think it is better to use CSS directly

**Comment by YongJieYongJie** [comment]
@tarun14110 Looks good!

**Comment by YongJieYongJie** [comment]
@tarun14110 Do help to update the branch and give me a ping!
(you can check the travis build log to find a timing where there are less activity to reduce the likelihood of the branch being out of date again)

**Comment by tarun14110** [comment]
@YongJieYongJie updated

**Pull request merged**